src/ekobox/cli/runcards.py

```diff
@@ -0,0 +1,39 @@
+"""Subcommand to manage runcards."""
+import logging
+import pathlib
+
+from .. import cards
+from . import library as lib
+from .base import command
+
+_logger = logging.getLogger(__name__)
+
+DESTINATION = pathlib.Path.cwd().absolute() / "runcards"
+"""Default destination for generated files"""
+
+option_dest = lib.destination(DESTINATION)
```

Comment:
isn't this a bit verbose, just for an additional option?

Reply:
this is copied from `nnusf`: this option is actually happening to be useful several times (not yet in EKO)